hdt-lib/include/Iterator.hpp

```diff
@@ -133,6 +133,8 @@ class IteratorTripleID {
 	}
 	virtual void goTo(unsigned int /*pos*/) {
 	}
```

Comment:
Should this one be private, since we cannot easily guess the offset?

Reply:
Having this public for e.g. random walk implementations can be useful. Also, hdt-it calls this function